src/view/treeNodes/fileChangeNode.ts

```diff
@@ -17,11 +17,12 @@ import { FileViewedDecorationProvider } from '../fileViewedDecorationProvider';
 import { DecorationProvider } from '../treeDecorationProvider';
 import { TreeNode, TreeNodeParent } from './treeNode';
 
-export function openFileCommand(uri: vscode.Uri): vscode.Command {
+export function openFileCommand(uri: vscode.Uri, options: Pick<vscode.TextDocumentShowOptions, 'preview'>): vscode.Command {
```

Comment:
Why not just use `vscode.TextDocumentShowOptions` instead of `Pick<vscode.TextDocumentShowOptions, 'preview'>`?